modules/nf-commons/src/test/nextflow/extension/BoltsTest.groovy

```diff
@@ -43,22 +46,24 @@ class BoltsTest extends Specification {
         def now = new Date(1513285947928)
 
         when:
-        def formatter = new SimpleDateFormat(fmt)
+        def formatter = new SimpleDateFormat(fmt, LocaleUtils.toLocale(locale))
         formatter.setTimeZone(TimeZone.getTimeZone(tz))
         then:
-        Bolts.format(now, fmt, tz) == expected
+        Bolts.format(now, fmt, tz, locale) == expected
         formatter.format(now) == expected
 
         where:
-        tz      | fmt                       | expected
-        'UTC'   | 'dd-MM-yyyy HH:mm'        | '14-12-2017 21:12'
-        'CET'   | 'dd-MM-yyyy HH:mm'        | '14-12-2017 22:12'
-        'UTC'   | 'dd-MMM-yyyy HH:mm:ss'    | '14-Dec-2017 21:12:27'
-        'CST'   | 'dd-MM-yyyy HH:mm'        | '14-12-2017 15:12'
-        'CST'   | 'dd-MMM-yyyy HH:mm:ss'    | '14-Dec-2017 15:12:27'
-
+        tz      | fmt                       | locale  | expected
+        'UTC'   | 'dd-MM-yyyy HH:mm'        | 'en'    | '14-12-2017 21:12'
+        'CET'   | 'dd-MM-yyyy HH:mm'        | 'en'    | '14-12-2017 22:12'
+        'UTC'   | 'dd-MMM-yyyy HH:mm:ss'    | 'en'    | '14-Dec-2017 21:12:27'
+        'CST'   | 'dd-MM-yyyy HH:mm'        | 'en'    | '14-12-2017 15:12'
+        'CST'   | 'dd-MMM-yyyy HH:mm:ss'    | 'en'    | '14-Dec-2017 15:12:27'
+        'UTC'   | 'dd-MMM-yyyy HH:mm:ss'    | 'es'    | '14-dic.-2017 21:12:27'
```

Comment:
My point is that it should be used always the English language irrespective of the current local. 

Therefore there's no need to specify the locale as 4th argument into `Bolts.format(now, fmt, tz, locale)`

Reply:
Something similar has been done here 

https://github.com/nextflow-io/nextflow/blob/462575f05bf3b9a97222320b83adf0d9c5a67d3d/modules/nf-httpfs/src/test/nextflow/file/http/XFileSystemProviderTest.groovy#L78-L93

Reply:
ah, ok, I misunderstood you

yes make sense

Reply:
It sets german as default locale (not the use of both `setLanguage` and `setRegion`. The test checks in any case `Fri` for the day of the week,